### PR TITLE
graphql: update dependency

### PR DESCRIPTION
**Pull request by viniciodf**
Update GraphQL Java to latest version (19.0), with important bugfixes.

### CODE REVIEW

**Comment by kingthorin** [comment]
Did you close this on purpose, or did it get closed because you used main branch and then reset the state?

**Comment by viniciodf** [comment]
I closed it because I didn't add "Signed-off-by" in the commit.
This is my first time opening a PR to zap-extensions.

File: addOns/graphql/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## Unreleased
 ### Changed
+- Update graphql-java dependency, with important bugfixes.
```
Comment (thc202) [inline addOns/graphql/CHANGELOG.md:8]: There's already an entry mentioning the update, should be changed that instead, though not sure the fix is that relevant for how the add-on is used.

**Comment by thc202** [comment]
You can amend the commit to add the signed-off.

**Comment by thc202** [comment]
The tests failed.

File: addOns/graphql/CHANGELOG.md (hunk 1 above)
Comment (viniciodf) [reply addOns/graphql/CHANGELOG.md:8]: An important fix was made in version 19.
According to the release notes:

https://github.com/graphql-java/graphql-java/releases/tag/v19.0

So when we import the graphql schema via URL, we get an invalid introspection query error due to the absence of the "isDeprecated" field.

Comment (thc202) [reply addOns/graphql/CHANGELOG.md:8]: All fixes are important (especially if they affect oneself), stating that does not adding anything relevant to the release notes (stating the issues fixed would but usually we don't add those details).

**Comment by thc202** [comment]
@viniciodf ping.

**Comment by kingthorin** [comment]
@viniciodf are you able to finish this?

**Comment by kingthorin** [comment]
Ping @viniciodf

**Comment by thc202** [comment]
Closing, no answer.

**Pull request closed without merging**